### PR TITLE
upload.sh: Fix some mistakes of labbots@5aa3fb5 | Fix labbots#57

### PR DESCRIPTION
**Pull request by Akianonymus**
* Fix  _get_latest_sha docs

* fix intendation in google-oauth2.sh

### CODE REVIEW

File: upload.sh [hunk 1]
```diff
@@ -525,13 +525,13 @@ _setup_arguments() {
                         ;;
                     root-dir)
                         _check_longoptions
-                        _check_default "${!OPTIND}" "ROOTDIR=${!OPTIND/default=/}" "UPDATE_DEFAULT_ROOTDIR=_update_config"
+                        ROOTDIR="${!OPTIND/default=/}"
+                        [[ ${!OPTIND} = default* ]] && UPDATE_DEFAULT_ROOTDIR="_update_config" || :
```
Comment (labbots) [inline upload.sh:529]: Codacy Issue found: [Note that A && B || C is not if-then-else. C may run when A is true.](https://app.codacy.com/manual/labbots/google-drive-upload/pullRequest?prid=5565059)

**Comment by labbots** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/labbots/google-drive-upload/pullRequest?prid=5565059&bid=18249674)

File: upload.sh [hunk 2]
```diff
@@ -607,10 +607,11 @@ _setup_arguments() {
                 FOLDERNAME="${OPTARG}"
                 ;;
             r)
-                _check_default "${OPTARG}" "ROOTDIR=${OPTARG/default=/}" "UPDATE_DEFAULT_ROOTDIR=_update_config"
+                ROOTDIR="${OPTARG/default=/}"
+                [[ ${OPTARG} = default* ]] && UPDATE_DEFAULT_ROOTDIR="_update_config" || :
```
Comment (labbots) [inline upload.sh:611]: Codacy Issue found: [Note that A && B || C is not if-then-else. C may run when A is true.](https://app.codacy.com/manual/labbots/google-drive-upload/pullRequest?prid=5565059)

**Comment by labbots** [comment]
@Akianonymus please be mindful of what you specify in the PR title and commit message. I don't condone offensive or harsh language in any communication within this community. I really appreciate all your contribution to the project but unacceptable language will not be tolerated.

**Comment by Akianonymus** [comment]
@labbots My apologies

Also, was a mistake.

**Comment by Akianonymus** [comment]
Also, I do appreciate your concern but before coming to a conclusion why i did what i did, please just ask nicely once. That was more like a threat 😂

ok, i am joking.

**Comment by labbots** [comment]
@Akianonymus looking at the commits. All the changes seems to yours. So mentioning that as mistakes of mine is misleading. So please do not mention your own mistakes to be of others.

**Comment by Akianonymus** [comment]
@labbots I was just referencing the commit the sha, because repo is in your account, it will show your name, doesn't it ??

Also, referencing commit sha means one is referencing the commit and the corresponding author, not the repo owner.

**Comment by labbots** [comment]
OK. I understand @Akianonymus Thanks for your explanation.

**Pull request merged**